### ISSUE TITLE
Expected format {org}/{repo}[/path]@ref. Actual 'nizarmah/auto-minify'

### ISSUE THREAD

**Issue by ardacebi** [nizarmah/auto-minify#7]
Hi,

I'm trying to use the simplest form on my project. Even though I'm doing everything right (at least for me), I'm getting this error:

`Expected format {org}/{repo}[/path]@ref. Actual 'nizarmah/auto-minify'`

My file:

```
name: CI

# Controls when the action will run. Triggers the workflow on push or pull request
# events but only for the master branch
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]

# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest

    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      # Checks-out your repository under $GITHUB_WORKSPACE, so auto-minify job can access it
       - uses: actions/checkout@v2

       - name: Auto Minify
         uses: nizarmah/auto-minify

       # Auto commits minified files to the repository
       # Ignore it if you don't want to commit the files to the repository 
       - name: Auto committing minified files
         uses: stefanzweifel/git-auto-commit-action@v3.0.0
         with:
           commit_message: "Automated: Minified CSS files"
           branch: ${{ github.ref }}

```

Thanks!

**Comment by nizarmah** [comment]
Hello ( : 

Alright, so there's something you need to add, it is the `version` of the GitHub Action.
So, it should look like this:
```
- name: Auto Minify
  uses: nizarmah/auto-minify@v1.6
```

Or you can use `@master` instead.

Let me know if there is anything else I can help with ( :

**Comment by ardacebi** [comment]
That did the trick, thanks!

**Comment by nizarmah** [comment]
@ardacebi you are most welcome ( :

if you run into any issues or if you have any feature requests, let me know by opening a new issue please :D 

Also, I updated the README on the Marketplace in order to avoid future confusions. If you wish to use the latest version, you can use `@master`!

**Comment by vivi90** [comment]
Dont works for me.
I get: `unable to find version master`

**Comment by nizarmah** [comment]
Hello @vivi90, there's a chance there have been some changes to how GitHub workflows work. I have been away from them for a while now.

However, I do recommend that you use `@v2.0`, since it's the latest tag. Utilizing a specific tag, instead of a branch, would help you ensure that the build would not break "unexpectedly". Accordingly, you can schedule upgrades to the package, later on, whenever necessary 👍🏼
